docs/source/docker/advanced/datasource_metadata.rst

```diff
@@ -13,17 +13,18 @@ Metadata Folder Structure
 
 Following, we outline the format in which metadata can be added to a
 Lightly datasource. Everything regarding metadata will take place in a subdirectory
-of your configured datasource called `.lightly/metadata`. The general structure
-of this directory will look like this:
-
+of your configured **output datasource** called `.lightly/metadata`. The general structure
```

Comment:
maybe put in a `(if configured otherwise input datasource)` ?

Reply:
we're now only showing how to configure the input and output datasource separately so I'd leave it like this :)